Makefile

```diff
@@ -22,12 +21,24 @@ run-api: build/keppel
 copy-fixtures:
 	find -name '*.actual' | xargs -I{} bash -c 'mv {} $$(echo {} | sed "s/.actual//g")'
 
-build-all: build/keppel
+privkey.pem:
+	openssl genrsa -out privkey.pem 4096
+
+OCI_DB_CMD := psql -U postgres -d keppel -h 127.0.0.1 -p 54321 -c "INSERT INTO accounts (name, auth_tenant_id) VALUES ('conformance-test', 'bogus') ON CONFLICT DO NOTHING;" -c "INSERT INTO quotas (auth_tenant_id, manifests) VALUES ('bogus', 100) ON CONFLICT DO NOTHING;"
+run-oci-conformance: build/keppel privkey.pem
+# little hack to populate the database
+	$(OCI_DB_CMD) || (set -euo pipefail && source ./.env-oci-conformance && timeout 4 $(CURDIR)/build/keppel server api || true) && $(OCI_DB_CMD)
+	@echo "Ready to run distribution-spec"
+	set -euo pipefail && source ./.env-oci-conformance && $(CURDIR)/build/keppel server api
+
+export KEPPEL_USERNAME=dummy-username
```

Comment:
Why is this needed in the Makefile when `.env-oci-conformance` overrides it?

Reply:
Oh, I'm guessing this is related to the usage of `DummyUserInfo` within the test auth driver? If so, that's spooky action at a distance and not good style. It would be much clearer if the `DummyUserInfo` carries the username in its own struct instead.